api/cedar_metadata_records/views.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import unicode_literals
+import logging
+
+from rest_framework import permissions as drf_permissions
+from rest_framework.exceptions import NotFound
+from rest_framework.generics import CreateAPIView, RetrieveAPIView, RetrieveUpdateDestroyAPIView
+from rest_framework.renderers import JSONRenderer
+from rest_framework.views import Response
+
+from api.base import permissions as base_permissions
+from api.base.parsers import (
+    JSONAPIMultipleRelationshipsParser,
+    JSONAPIMultipleRelationshipsParserForRegularJSON,
+)
+from api.base.versioning import PrivateVersioning
+from api.base.views import JSONAPIBaseView
+from api.cedar_metadata_records.permissions import CedarMetadataRecordPermission
+from api.cedar_metadata_records.serializers import (
+    CedarMetadataRecordsCreateSerializer,
+    CedarMetadataRecordsDetailSerializer,
+)
+from framework.auth.oauth_scopes import CoreScopes
+
+from osf.models import CedarMetadataRecord
+
+logger = logging.getLogger(__name__)
+
+
+class CedarMetadataRecordList(JSONAPIBaseView, CreateAPIView):
+
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+    )
+    required_read_scopes = [CoreScopes.CEDAR_METADATA_RECORD_READ]
```

Comment:
```suggestion
    required_read_scopes = [CoreScopes.NULL]
```
Minor: This view can't be read from; it should probably use `CoreScopes.NULL` here.

Reply:
Update: this doesn't affect cookie or basic auth. However, for token auth, this has a side effect of returning "permission denied" instead of "method not allowed". My guess is that scope is checked before request method.